commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -170,7 +170,8 @@ def get_customer_by_lms_user_id(self, lms_user_id: int) -> Optional[CTCustomer]:
             return results.results[0]
 
     def get_orders(self, customer: CTCustomer, offset=0,
-                   limit=ORDER_HISTORY_PER_SYSTEM_REQ_LIMIT) -> PaginatedResult[CTOrder]:
+                   limit=ORDER_HISTORY_PER_SYSTEM_REQ_LIMIT,
+                   where="orderState=\"Complete\"") -> PaginatedResult[CTOrder]:
```

Comment:
If we are making the order state a param then I would suggest we instead just pass `order_state` param with default value of `Complete` and format it inside the `get_orders` function. This would make calling this utility easier on the part of user and we can do the remaining formatting inside the utility.